website/content/docs/k8s/deployment-configurations/argo-rollouts-configuration.mdx

```diff
@@ -0,0 +1,221 @@
+        location: "file:///plugin-bin/hashicorp/rollouts-plugin-trafficrouter-consul"
+  volumes:
+    - name: consul-plugin
+      emptyDir: {}
+  volumeMounts:
+    - name: consul-plugin
+      mountPath: /plugin-bin/hashicorp
+```
+```bash
+helm install argo-rollouts argo/argo-rollouts -f values.yaml -n argo-rollouts
+```
+
+### Install Rollouts Using Helm (Binary)
+
+1. Build this plugin (i.e. `make build`).
+2. Mount the built plugin on to the `argo-rollouts` container
+3. Add the following code to your `values.yaml` file, then install the argo-rollouts by helm:
+
+```yaml
+  trafficRouterPlugins:
+    trafficRouterPlugins: |-
+      - name: "argoproj-labs/consul"
+        location: "file:///plugin-bin/hashicorp/rollouts-plugin-trafficrouter-consul"
+  volumes:
+    - name: consul-route-plugin
+      hostPath:
+        # The path being mounted to, change this depending on your mount path
+        path: /rollouts-plugin-trafficrouter-consul
+        type: DirectoryOrCreate
+  volumeMounts:
+    - name: consul-route-plugin
+      mountPath: /plugin-bin/hashicorp
+```
+```bash
+helm install argo-rollouts argo/argo-rollouts -f values.yaml -n argo-rollouts
+```
+
+### Stand-alone installation
+
+NOTES:
+**_1. The file as follows (and the codes in it) just for illustrative purposes only, please do not use directly!_**
+
+Steps:
+
+1. Build this plugin.
+2. Put the plugin on the path & mount onto the `argo-rollouts`container (please refer to the example YAML below to modify the deployment)
+3. Create a ConfigMap to configure `argo-rollouts` with the plugin's location:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: argo-rollouts
+  namespace: argo-rollouts
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: argo-rollouts
+  template:
+    spec:
+      # ...
+      volumes:
+        # ...
+         - name: consul-plugin
+           hostPath:
+             path: /plugin-bin/hashicorp/rollouts-plugin-trafficrouter-consul
+             type: DirectoryOrCreate
+      containers:
+        - name: argo-rollouts
+        # ...
+          volumeMounts:
+             - name: consul-route-plugin
+               mountPath: /plugin-bin/hashicorp
+
+```
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argo-rollouts-config
+  namespace: argo-rollouts
+data:
+  trafficRouterPlugins: |-
+    - name: "argoproj-labs/consul"
+      location: "file:///plugin-bin/hashicorp/rollouts-plugin-trafficrouter-consul"
+binaryData: {}
+```
+
+### Install the RBAC
+
+After either mounting the binary or using an init container apply the RBAC using the provided `yaml/rbac.yaml`
+```bash
+kubectl apply -f https://raw.githubusercontent.com/argoproj-labs/rollouts-plugin-trafficrouter-consul/main/yaml/rbac.yaml
+```
+
+## Usage
+
+1. Create the Kubernetes Service that will be utilized by the service being rolled out. Additionally, configure any Service Defaults and Proxy Defaults required for the service.
```

Comment:
Using `1.` each time isn't working as intended when the site renders. I know markdown rendering usually handles this, but each numbered item is coming up as `1.`